### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by IjayAbby**
## Project Requirements.

-  IMPORTANT NOTE This is the first time you will use Gitflow. Keep that in mind!
-  Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise).
- Use ES6 modules.

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Invalid submission 

Hi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with the wrong Gitflow. You are supposed to create a Pull Request that is from `feature` to `development`, not `main` branch. Please create a new pull request and submit a link of the Pull request instead of https://github.com/IjayAbby/leaderboard-api/pulls/

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          <h2>Add your score</h2>
+          <div class="input-scores">
+            <input id="user-name" name="user-name" placeholder="Your name" required />
+            <input id="user-score" name="user-score" placeholder="Your score" required />
+            <input id="user-data-submit" type="submit" value="Submit" />
+          </div>
+        </article>
+    </div>
+</body>
+</html>
```
Comment (arslanbisharat) [inline src/index.html:29]: - [ ] There is no need for two index.html files. index.html file should be present only in the dist folder. so I would recommend you remove this one. 😄

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.html (hunk 1 above)
Comment (IjayAbby) [reply src/index.html:29]: The /dist folder contains the minimized version of the source code. The code present in the /dist folder is actually the code which is used on production web applications. Along with the minified code, the /dist folder also comprises of all the compiled modules that may or may not be used with other systems.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

After talking to SCR, I think you can have index.html twice.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**